App.js

```diff
@@ -28,13 +28,23 @@ export default class App extends Component<{}> {
   requestForm() {
     usabilla.loadFeedbackForm("YOUR_FORM_ID_HERE")
   }
+  resetCampaigns() {
+    usabilla.resetCampaigns()
+  }
```

Comment:
missing new lines between methods?

Reply:
So far we kept them one after the other without spaces

Reply:
is it a js convention?

Reply:
not really... checking other projects it seems they have spaces so spaces will be added